site/static/api/device-registry-v1.yaml

```diff
@@ -519,6 +526,12 @@ components:
             "enabled":
                type: boolean
                default: true
+            "defaults":
+               $ref: '#/components/schemas/Settings'
+            "via":
+               type: string
```

Comment:
this should be an array of Strings as you can allow a device to connect via any one of multiple gateways ...

Reply:
Are you sure about that? Because in the `device-identities.json` used for testing it is only a `String`.

Reply:
Yes, absolutely sure. The example registry allows to read a single String for backwards compatibility reasons only. However, the external API already uses an array ...

Reply:
Updated.